uptane-standard.md

```diff
@@ -732,12 +741,13 @@ For secondaries without storage, the primary SHOULD wait for a request from the
 
 ### Installing images on primary or secondary ECUs
 
-Before installing a new image, an ECU SHALL perform the following five steps:
+An ECU SHALL perform the following steps when attempting to install a new image:
 
 1. Verify latest attested time ({{verify_time}})
 1. Verify metadata ({{verify_metadata}})
 1. Download latest image ({{download_image}})
 1. Verify image ({{verify_image}})
+1. Install image ({{install_image}})
 1. Create and send version report ({{create_version_report}})
```

Comment:
```suggestion
6. Create and send version report ({{create_version_report}})
```

Reply:
As commented in other PR, it's better to avoid explicit numbers to simplify further changes - markdown renderer will take care of this automatically.